tests/manage/pv_services/test_ceph_daemon_kill_during_resource_deletion.py

```diff
@@ -300,7 +308,7 @@ def test_ceph_daemon_kill_during_pod_pvc_deletion(
         # Verify PVs are deleted
         for pv_obj in pv_objs:
             assert pv_obj.ocp.wait_for_delete(
-                pv_obj.name, 120
+                pv_obj.name, 600
```

Comment:
this is a significant increase in timeout. Why is it needed?

Reply:
The test case is failing in ROSA platform due to timeout. 5-8 minutes delete time is observed in recent runs. In different runs so far we have seen intermittent failure across different platforms due to the timeout error.  The PV  will eventually delete.

Reply:
so maybe we can increase the timeout only for ROSA and also, that should be considered a workround

Reply:
The issue is closed. Reverted this code change.